### PR TITLE
[Doc-Only]Add Error code handling documentation and split project information

### PR DESCRIPTION
**Pull request by cjyclaire**
Address issue #1060 and aws/aws-php-sns-message-validator#15

cc:\ @xibz @mtdowling @petermoon

### CODE REVIEW

File: docs/faq.rst [hunk 1]
```diff
@@ -253,3 +253,34 @@ and ``BandwidthLimitExceeded`` are handled with retries as well.
 
 SDK also integrates exponential delay with backoff and jitter algorithm in retry scheme. Furthermore,
 default retry behavior is configured as ``3`` for all services except dynamoDB, which is ``10``.
+
+How to handle exception with Error code?
+----------------------------------------
+
+Besides Exception class customized in SDK, Each Aws Service Client has its own exception class which is
```
Comment (deviantintegral) [inline docs/faq.rst]: Minor grammar / casing here, how about:

> Besides AwsException, each AWS service client has it's own exception class that inherits from [snip]

File: docs/faq.rst [hunk 2]
```diff
@@ -253,3 +253,34 @@ and ``BandwidthLimitExceeded`` are handled with retries as well.
...
+inherited from `Aws\Exception\AwsException <http://docs.aws.amazon.com/aws-sdk-php/v3/api/class-Aws.Exception.AwsException.html>`_.
+With Api specific Errors listed under ``Errors`` section of each method, you can determine more specific error type to
```
Comment (deviantintegral) [inline docs/faq.rst]: >With the API specific errors listed under the ``Errors`` section of each method, you can determine more specific error types to catch.

Comment (deviantintegral) [inline docs/faq.rst]: Actually to not be passive:

> You can determine more specific error types to catch with the API specific errors listed under the ``Errors`` section of each method.

**Comment by deviantintegral** [comment]
Thanks for the docs update! This would have answered my questions. Just some super minor wording suggestions.

**Comment by cjyclaire** [comment]
@deviantintegral Thanks for the suggestion! I'll make changes in another commit.

**Pull request merged**